### PR TITLE
feat(starfish): Shows action and domain columns in span view

### PR DESCRIPTION
**Pull request by edwardgou-sentry**
Shows action and domain columns in generic and module span views to match filter options

### CODE REVIEW

File: static/app/views/starfish/views/spans/spansTable.tsx [hunk 1]
```diff
@@ -197,10 +198,19 @@ function getDescriptionHeader(moduleName: ModuleName) {
   }
   return 'Description';
 }
+function getActionHeader(moduleName: ModuleName) {
+  if (moduleName === ModuleName.HTTP) {
+    return 'HTTP Method';
+  }
+  if (moduleName === ModuleName.DB) {
+    return 'SQL Command';
```
Comment (gggritso) [inline static/app/views/starfish/views/spans/spansTable.tsx:206]:
```suggestion
    return t('SQL Command');
```

**Review by gggritso — COMMENTED** [review]
Hm, would you mind quickly asking Dora/Product about this? Maybe it is a little strange that there is an "Action" column without an "Action" selector, but it's _even more strange_ to have these completely redundant columns:

Screenshot 2023-07-10 at 3 19 22 PM
Screenshot 2023-07-10 at 3 19 41 PM

**Comment by edwardgou-sentry** [comment]
> Hm, would you mind quickly asking Dora/Product about this? Maybe it is a little strange that there is an "Action" column without an "Action" selector, but it's _even more strange_ to have these completely redundant columns:
> 
> Screenshot 2023-07-10 at 3 19 22 PM Screenshot 2023-07-10 at 3 19 41 PM

good point! Reached out!

**Comment by edwardgou-sentry** [comment]
@gggritso going to close this for now. Some of the data on actions and domain are not nailed down for generic spans (especially redis) so we're not really sure whether to show or hide these columns yet.

**Pull request closed without merging**